modules/.htaccess

```diff
@@ -1,4 +1,4 @@
-<FilesMatch "(\.tpl|\.twig|config\.xml|config_([a-z]+)\.xml)$">
+<FilesMatch "(\.php|\.log|\.txt|\.lock|\.json|\.yml|\.yaml|\.tpl|\.twig|config\.xml|config_([a-z]+)\.xml)$">
```

Comment:
I agree with the general idea, but I think some conditions are a bit overkill

Especially json and txt files shouldn't be blocked IMHO:
- json files may be used as "static API"
- txt files can be used as assets, and I don't see much risk that a .txt files would contain sensitive info or config

Although it would make sense to block composer files but it could be explicitly blocking `composer.json` and `composer.lock` only

Reply:
```suggestion
<FilesMatch "(\.php|\.log|\.lock|composer\.json|\.yml|\.yaml|\.tpl|\.twig|config\.xml|config_([a-z]+)\.xml)$">
```

Reply:
Ah also after reading @ShaiMagal comments, although I understand he's not particularly in favor of this modification, I think blocking SQL files also makes sense 🤔

Reply:
> json files may be used as "static API"
Should be served by a front controller after verifying some authorization.
What about packages.json witch can display npm depenciencies ?

Reply:
Hi, I also think that all JSON content and other data files like xml or txt or csv should be served by front controllers

Reply:
@LouisAUTHIE What security benefit will serving these files over a controller, without any security token bring?